LIMS2DB/classes.py

```diff
@@ -655,21 +654,29 @@ def save(self, update_modification_time=True):
                 db.save(self.obj)
                 if self.obj.get("details", {}).get("type", "") == "Application":
                     lib_method_text = f"Library method: {self.obj['details'].get('library_construction_method', 'N/A')}"
+                    application = self.obj.get('details', {}).get('application', '')
+                    is_single_cell =  application == 'RNA-seq (single cell)'
+                    if is_single_cell:
+                        single_cell_text = f"[Application: {application}]"
```

Comment:
You already now the result of the f-string since application only has one allowed value in this if-case...

Reply:
We could add more applications to check for in the future.